package com.google.android.gms.internal.ads;

/* compiled from: com.google.android.gms:play-services-ads@@24.5.0 */
public final class zzwo implements zzvg {
    private final zzgi zza;
    private int zzb = 1048576;
    private final zzwn zzc;
    private final zzzq zzd;

    public zzwo(zzgi zzgi, zzwn zzwn) {
        zzzq zzzq = new zzzq(-1);
        this.zza = zzgi;
        this.zzc = zzwn;
        this.zzd = zzzq;
    }

    public final zzwo zza(int i) {
        this.zzb = i;
        return this;
    }

    public final zzwq zzb(zzap zzap) {
        zzap.zzb.getClass();
        return new zzwq(zzap, this.zza, this.zzc, zzsh.zza, this.zzd, this.zzb, 0, (zzz) null, (zzfwh) null, (zzwp) null);
    }
}
